<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1062</width>
    <height>781</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>1031</width>
      <height>821</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Diff</string>
     </attribute>
     <widget class="QScrollArea" name="scrollArea">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>851</width>
        <height>191</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>849</width>
         <height>189</height>
        </rect>
       </property>
       <widget class="QLabel" name="av1">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>831</width>
          <height>171</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QScrollArea" name="scrollArea_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>851</width>
        <height>191</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>849</width>
         <height>189</height>
        </rect>
       </property>
       <widget class="QLabel" name="av2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>831</width>
          <height>171</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QScrollArea" name="scrollArea_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>400</y>
        <width>851</width>
        <height>171</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_4">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>849</width>
         <height>169</height>
        </rect>
       </property>
       <widget class="QLabel" name="avFinal">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>831</width>
          <height>151</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>600</y>
        <width>114</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Processar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="v1_diff_load">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>80</y>
        <width>41</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QPushButton" name="v2_diff_load">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>290</y>
        <width>41</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QPushButton" name="delta_save">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>600</y>
        <width>71</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
     <widget class="QSlider" name="slider">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>580</y>
        <width>891</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Patch</string>
     </attribute>
     <widget class="QPushButton" name="btnPatchProcess">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>650</y>
        <width>114</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Processar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnLoadV1">
      <property name="geometry">
       <rect>
        <x>890</x>
        <y>90</y>
        <width>41</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnDiffLoad">
      <property name="geometry">
       <rect>
        <x>880</x>
        <y>500</y>
        <width>41</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QSlider" name="patch_slider">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>630</y>
        <width>971</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QScrollArea" name="scrollArea_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>851</width>
        <height>191</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>849</width>
         <height>189</height>
        </rect>
       </property>
       <widget class="QLabel" name="lblV1_to_Patch">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>831</width>
          <height>171</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QScrollArea" name="scrollArea_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>851</width>
        <height>191</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_8">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>849</width>
         <height>189</height>
        </rect>
       </property>
       <widget class="QLabel" name="lblDiff">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>831</width>
          <height>171</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QScrollArea" name="scrollArea_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>430</y>
        <width>851</width>
        <height>191</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_10">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>849</width>
         <height>189</height>
        </rect>
       </property>
       <widget class="QLabel" name="lblPatchResult">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>831</width>
          <height>171</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QPushButton" name="btn_saveVideoPatched">
      <property name="geometry">
       <rect>
        <x>880</x>
        <y>310</y>
        <width>41</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Merge</string>
     </attribute>
     <widget class="QSlider" name="merge_slider">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>620</y>
        <width>891</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="btnProcessMerge">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>650</y>
        <width>114</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Processar</string>
      </property>
     </widget>
     <widget class="QScrollArea" name="scrollArea_6">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>10</y>
        <width>471</width>
        <height>161</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_6">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>469</width>
         <height>159</height>
        </rect>
       </property>
       <widget class="QLabel" name="lblBaseMerge">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>441</width>
          <height>141</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QPushButton" name="base_merge_load">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>180</y>
        <width>41</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QPushButton" name="v1_merge_load">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>400</y>
        <width>41</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QScrollArea" name="scrollArea_7">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>230</y>
        <width>471</width>
        <height>161</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_11">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>469</width>
         <height>159</height>
        </rect>
       </property>
       <widget class="QLabel" name="lblV1Merge">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>441</width>
          <height>141</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QScrollArea" name="scrollArea_8">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>230</y>
        <width>471</width>
        <height>161</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_13">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>469</width>
         <height>159</height>
        </rect>
       </property>
       <widget class="QLabel" name="lblV2Merge">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>441</width>
          <height>141</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QScrollArea" name="scrollArea_9">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>440</y>
        <width>471</width>
        <height>161</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_15">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>469</width>
         <height>159</height>
        </rect>
       </property>
       <widget class="QLabel" name="lblResultMerge">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>441</width>
          <height>141</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QPushButton" name="v2_merge_load">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>400</y>
        <width>41</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1062</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
